backend/Dockerfile

```
@@ -1,12 +1,31 @@
+COPY build.gradle.kts settings.gradle.kts ./
+COPY config config
+COPY src src
+
+RUN chmod +x gradlew && ./gradlew bootJar --no-daemon
+
 FROM eclipse-temurin:21-jre
 
 WORKDIR /app
 
-COPY build/libs/backend-*.jar app.jar
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)